service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/utils/Verifiers.java

```diff
@@ -0,0 +1,35 @@
+package uk.nhs.adaptors.oneoneone.cda.report.mapper.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.hl7.fhir.dstu3.model.ContactPoint;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.hl7.fhir.dstu3.model.Period;
+
+public class Verifiers {
```

Comment:
@dawidskainos, should we have separate verifier class for each mapper then, how do you think? Or should not we keep the verifiers in separate class?

I moved the into separate class because such common mappers will be used in many components or so tested in several places.

Reply:
If they're going to be reused in other tests - that's ok, we can keep them here. But when you turn mappers into spring components then probably we won't have to use them in other tests as they'll be mocked.

Reply:
moved to Encounter Test, yes, we will use mock in other tests